config/jobs/ocp-power-automation/ocp-power-automation.github.io/ocp-power-automation.github.io-postsubmits.yaml

```diff
@@ -0,0 +1,32 @@
+postsubmits:
+  ocp-power-automation/ocp-power-automation.github.io:
+    - name: pull-github-create-page
+      decorate: true
+      always_run: true
+      branches:
+        - ^main$
+      skip_submodules: false
+      spec:
+        volumes:
+          - name: github-key
+            secret:
+              secretName: github-key-secret
```

Comment:
I just named the secret key as: `ocp-gh-deploykey`, can you please change the name here accordingly?